modules/api-gateway/main.tf

```diff
@@ -61,6 +62,7 @@ resource "aws_api_gateway_method" "orca_catalog_reporting_api_method" {
   resource_id   = aws_api_gateway_resource.orca_catalog_reporting_api_resource_catalog_reconcile.id
   http_method   = "POST"
   authorization = "NONE"
+  api_key_required = false
```

Comment:
It's optional, KICS flags it as a vulnerability since it's not specified (defaults to false). We can remove it until we research and decide if we want to use API Keys.

Reply:
Sounds good.